examples/podinfo/BUILD.bazel

```diff
@@ -0,0 +1,64 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//tools:cdk8s.bzl", "cdk8s_import", "cdk8s_synth")
+load("//tools:defaults.bzl", "jest_test", "ts_library")
+
+cdk8s_import(
+    name = "gen",
+    include = "io.k8s.api.autoscaling.v2beta2.HorizontalPodAutoscaler",
+    path = "examples/podinfo/imports",
+    language = "typescript"
+)
+
+ts_library(
+    name = "lib",
+    srcs = glob(
+        ["**/*.ts"],
+        exclude = ["**/*.test.ts"],
+    ),
+    deps = [
+        "//packages/cdk8s:lib",
+        "@npm//@aws-cdk/core",
+    ],
+)
+
+cdk8s_synth(
+    name = "synth",
+    path = "dist",
+    data = [":lib"],
+    app = "TODO"
+)
+
+ts_library(
+    name = "tests",
+    testonly = True,
+    srcs = glob(["**/*.test.ts"]),
+    deps = [
+        ":lib",
+    ],
+)
+
+filegroup(
+    name = "test_sources",
+    testonly = True,
+    srcs = [":tests"],
+    output_group = "es5_sources",
+)
+
+filegroup(
+    name = "sources",
+    testonly = True,
+    srcs = [":lib"],
+    output_group = "es5_sources",
+)
+
+jest_test(
+    name = "test",
+    srcs = glob(["**/*.snap"]) + [
+        ":test_sources",
+        ":sources",
```

Comment:
same note, this whole file needs to be a single line

Reply:
Agreed, will add macro.